airbyte-integrations/connectors/source-looker/unit_tests/test_URL.py

```diff
@@ -0,0 +1,44 @@
+import re
+import unittest
+
+def valid_URL(test_str):

```

Comment:
do you how we can avoid duplication and use regex from spec file here?

Reply:
I think may be we can access spec dict and can refer pattern using spec["properties"]["domain"]["pattern"]. I'm quite unsure of the imports here. can you point me an example here

Reply:
try to search `@pytest.fixture` in our codebase, there will be plenty of examples